### PR TITLE
adding role bindings feature. Fixes #55

### LINKED ISSUE #55: Add support for Role bindings
Add support for Role bindings

### PR DESCRIPTION
**Pull request by shinusuresh**

### CODE REVIEW

File: src/ui/resource_tabs.rs [hunk 1]
```diff
@@ -36,6 +36,7 @@ static SECRETS_TITLE: &str = "Secrets";
 static RPL_CTRL_TITLE: &str = "ReplicationControllers";
 static STORAGE_CLASSES_LABEL: &str = "StorageClasses";
 static ROLES_TITLE: &str = "Roles";
+static ROLE_BINDINGS_TITLE: &str = "Role bindings";
```
Comment (deepu105) [inline src/ui/resource_tabs.rs:39]:
```suggestion
static ROLE_BINDINGS_TITLE: &str = "RoleBindings";
```

**Review by deepu105 — APPROVED** [review]

**Comment by shinusuresh** [comment]
👍

File: src/ui/resource_tabs.rs [hunk 2]
```diff
@@ -1062,6 +1065,59 @@ fn draw_roles_block<B: Backend>(f: &mut Frame<'_, B>, app: &mut App, area: Rect)
...
+  );
+
+  draw_resource_block(
+    f,
+    area,
+    ResourceTableProps {
+      title,
+      inline_help: DESCRIBE_YAML_AND_ESC_HINT.into(),
+      resource: &mut app.data.role_bindings,
+      table_headers: vec!["Name", "Role", "Age"],
```
Comment (deepu105) [inline src/ui/resource_tabs.rs:1101]:
```suggestion
      table_headers: vec!["Namespace", "Name", "Role", "Age"],
```

File: src/ui/resource_tabs.rs [hunk 3]
```diff
@@ -1062,6 +1065,59 @@ fn draw_roles_block<B: Backend>(f: &mut Frame<'_, B>, app: &mut App, area: Rect)
...
+      column_widths: vec![
+        Constraint::Percentage(40),
```
Comment (deepu105) [inline src/ui/resource_tabs.rs:1103]:
```suggestion
        Constraint::Percentage(20),
        Constraint::Percentage(30),
```

File: src/ui/resource_tabs.rs [hunk 4]
```diff
@@ -1062,6 +1065,59 @@ fn draw_roles_block<B: Backend>(f: &mut Frame<'_, B>, app: &mut App, area: Rect)
...
+        Constraint::Percentage(40),
```
Comment (deepu105) [inline src/ui/resource_tabs.rs:1104]:
```suggestion
        Constraint::Percentage(30),
```

**Comment by deepu105** [comment]
Thank you @shinusuresh

**Comment by shinusuresh** [comment]
👍

**Pull request merged**